WireAPI/Sources/WireAPI/Network/APIService/APIService.swift

```diff
@@ -53,6 +53,7 @@ public final class APIService: APIServiceProtocol {
     private let backendWebSocketURL: URL
```

Comment:
do we still need to have this property in the `APIService` now that we have a dedicated `PushChannelService` which is using the websocket ?

Reply:
nope, I'll remove it.